.github/workflows/project-automation.yml

```diff
@@ -0,0 +1,14 @@
+name: project-automation
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  add-to-project:
+    name: Add new issues to the core/app project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/celestiaorg/projects/24
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
if we add this, we need an admin on this repo to ensure this secret exists

Reply:
This is also an organization wide secret

Reply:
Great! Resolving.